### DISCUSSION TITLE
v3.5.1

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#324] [Announcements]
### New Features

- Enhance the stability when viewing channel details (BiliBili)

### Bug Fixes

- Random 403 error at 1 min (YouTube)
- Errors are thrown if the channel doesn't have a banner (YouTube)
- Live status is always shown as "投稿视频轮播" (BiliBili)
- No thumbnail on some old android devices
- Full screen button don't rotate the screen to landscape on tablets
- Some unexpected error under certain circumstances

### UI

- Add a play queue button for easier access to background queue
- Show the last update time of the error solution
- Update tv banner

<hr/>
If you're enjoying PipePipe, please help share it with your friends. Every bit of support means a lot to us!
<hr/>

### Which APK should you download?

- arm64-v8a should work on most devices.
- armeabi-v7a if your device is a 10+ year old phone.
- x86_64 for 64-bit Android simulators (on a PC).
- x86 for 32-bit simulators.

If these rules not working, download the universal APK. It should work on any device.
All APKs provide the same contents and functions, but the universal one may be slower than the split ones.

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1308/PipePipe/releases/tag/v3.5.1'>v3.5.1</a>.</em>

**Comment by jojojojojoj5564656465465** [comment]
Can I import my settings from Newpipe ? i have playlist
